indico/web/forms/fields/files.py

```diff
@@ -0,0 +1,65 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals, absolute_import
+
+from wtforms import Field
+
+from indico.core.config import Config
+from indico.web.forms.widgets import JinjaWidget
+
+
+class FileField(Field):
+    """A dropzone field"""
+
+    widget = JinjaWidget('forms/dropzone_widget.html')
+
+    default_options = {
+        'max_files': 10,
+        'add_remove_links': True,
+        'param_name': 'file',
```

Comment:
Are you sure this is the right thing to do atm? It will require changes in all the controllers where the `FileField` is used, many of them use mixins and base classes to handle the file uploads